app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+
+  belongs_to :user
+
+  def increase_counter
+    user.increment!(:post_counter)
+  end
```

Comment:
- [ ] The requirements say that the post should be updated. `def update_post_counter` and also it would be better if it's private.

![Capture web_7-12-2021_104450_github com](https://user-images.githubusercontent.com/61562626/145005894-ee1ab16f-1b40-4010-bb88-9002125c5ce8.jpeg)